dash/quorum/validator_conn_executor.go

```diff
@@ -256,7 +265,7 @@ func (vc *ValidatorConnExecutor) disconnectValidators(exceptions validatorMap) e
 // to be established. It will also disconnect previous validators.
 func (vc *ValidatorConnExecutor) updateConnections() error {
 	// We only do something if we are part of new ValidatorSet
-	if _, ok := vc.validatorSetMembers[vc.nodeID]; !ok {
+	if me := vc.me(); me == nil {
```

Comment:
checking on existence, would be better to place in a separate function `hasCurrInSet() bool` or `hasMeInSet() bool`